pkg/querier/tripperware/query.proto

```diff
@@ -93,6 +93,7 @@ message PrometheusQueryResult {
     Vector vector = 1;
     bytes rawBytes = 2;
     Matrix matrix = 3;
+    Scalar scalar = 4;
```

Comment:
Do we need a dedicated `scalar` type here? `rawBytes` should already cover `scalar` data type

Reply:
The reason is to explicitly express the scalar type and then use it in the ruler decode logic.
Should we sustain the existing query.proto?

Reply:
Let's remove that I think... It shouldn't be too hard to decode that using raw bytes